bot.js

```diff
@@ -61,16 +62,17 @@ client.on('message', message => {
 	}
 
 	if (message.content === `${prefix}groups`) {
-		const group1 = formatGroupMessage("Group 1", groups[0] || pool)
-		const group2 = formatGroupMessage("Group 2", groups[1])
-		const waitlistGroup = formatGroupMessage("Waitlist", waitlist)
 		const differenceMsg = formatDifferenceMessage(computeDifference(groups[0], groups[1], true))
-		let msg = `Group 1: ${group1} (${group1.length})`
+		const chaosPinkBeanGroup = assemblePinkBeanGroup([...leaders, ...pool])
+		let msg = `Group 1: ${formatGroupMessage(groups[0] || leaders)} (${totalRank(groups[0] || leaders)})`
```

Comment:
Was thinking `msg` would be an array, and you would just push things into it, and join("\n") in the end. But I could do that in refactor if you want to exclude that as part of this commit

Reply:
Yeah I was thinking the same 👍